sentinel-core/src/test/java/com/alibaba/csp/sentinel/util/VersionUtilTest.java

```diff
@@ -27,4 +29,19 @@ public void testGetDefaultVersion() {
         // Manifest cannot be load before package.
         Assert.assertEquals(defaultVersion, version);
     }
+    
+    @Test
+    public void testFromVersionString() {
+        assertEquals(0x01020300, VersionUtil.fromVersionString("1.2.3"));
+        assertEquals(0x01020304, VersionUtil.fromVersionString("1.2.3.4"));
+        assertEquals(0x0102ff04, VersionUtil.fromVersionString("1.2.255.4"));
+        assertEquals(0xffffffff, VersionUtil.fromVersionString("255.255.255.255"));
+        assertEquals(0, VersionUtil.fromVersionString("1.255.256.0"));
+        assertEquals(0, VersionUtil.fromVersionString("1.2."));
+        assertEquals(0x01000000, VersionUtil.fromVersionString("1"));
+        assertEquals(0x01020000, VersionUtil.fromVersionString("1.2"));
+        assertEquals(0, VersionUtil.fromVersionString("test"));
+        assertEquals(0, VersionUtil.fromVersionString("1.2.3-"));
+        assertEquals(0, VersionUtil.fromVersionString("1.2.3b"));
+    }
```

Comment:
I found that `fastjson` has versions like `1.2.60.sec09`
[https://mvnrepository.com/artifact/com.alibaba/fastjson](https://mvnrepository.com/artifact/com.alibaba/fastjson)

Do we need to handle with that...